safe/report/test/test_impact_report.py

```diff
@@ -1063,6 +1065,152 @@ def test_minimum_needs_outputs(self):
 
         shutil.rmtree(output_folder, ignore_errors=True)
 
+    def test_minimum_needs_outputs_modified(self):
+        """Test generate minimum needs section with updated profile
```

Comment:
The regex is not able to read multiline comments to add the full stop. but it's test file, I don't mind ;-)